easybuild/easyconfigs/r/RELION/RELION-1.4-intel-2016b-single.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'RELION'
+version = '1.4'
+versionsuffix = '-single'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'opt': True}
```

Comment:
Ah, yes, I overlooked that, sorry.

Any particular reason you really want `-O3` here? If so, please add a comment in the easyconfig to make that clear.